src/pages/Explore/index.tsx

```diff
@@ -110,40 +110,56 @@ const SearchContainer = styled(FiltersContainer)<{ isInfoExplorePageEnabled: boo
     ${({ isInfoExplorePageEnabled }) => isInfoExplorePageEnabled && 'justify-content: flex-end;'}
   }
 `
+export enum ExploreTab {
+  Tokens = 'tokens',
+  Pools = 'pools',
+  Transactions = 'transactions',
+}
 
 interface Page {
   title: React.ReactNode
-  key: string
+  key: ExploreTab
   component: () => JSX.Element
   loggingElementName: string
 }
 const Pages: Array<Page> = [
   {
     title: <Trans>Tokens</Trans>,
-    key: 'tokens',
+    key: ExploreTab.Tokens,
     component: TokenTable,
     loggingElementName: InterfaceElementName.EXPLORE_TOKENS_TAB,
   },
   {
     title: <Trans>Pools</Trans>,
-    key: 'pools',
+    key: ExploreTab.Pools,
     component: TokenTable,
     loggingElementName: InterfaceElementName.EXPLORE_POOLS_TAB,
   },
   {
     title: <Trans>Transactions</Trans>,
-    key: 'transactions',
+    key: ExploreTab.Transactions,
     component: TokenTable,
     loggingElementName: InterfaceElementName.EXPLORE_TRANSACTIONS_TAB,
   },
 ]
 
-const Explore = () => {
+const Explore = ({ initialTab }: { initialTab?: ExploreTab }) => {
   const resetFilterString = useResetAtom(filterStringAtom)
   const location = useLocation()
-  const [currentTab, setCurrentTab] = useState(0)
+  const navigate = useNavigate()
+
+  const initialKey = useMemo(() => initialTab && Object.values(ExploreTab).indexOf(initialTab), [initialTab])
+  const [currentTab, setCurrentTab] = useState(initialKey ?? 0)
   const isInfoExplorePageEnabled = useInfoExplorePageEnabled()
 
+  useEffect(() => {
+    const tabFromPath = location.pathname.split('/')[2] // assuming the tab is the third part of the URL
```

Comment:
instead of making this assumption, could we use the `useParams` hook to guarantee we get the correct field?